.changeset/strange-moose-clean.md

```diff
@@ -0,0 +1,5 @@
+---
+'@graphql-codegen/schema-ast': major
```

Comment:
```suggestion
'@graphql-codegen/schema-ast': minor
```

Reply:
Thank you! I wasn't sure on this one because technically nothing was broken, but also doesn't really have any breaking changes.